README.md

```diff
@@ -8,7 +8,22 @@ mod-panel is a simple control panel to start MOD Host + UI and SDK services.
 
 ## Usage
 
-Simply clone this repository recursively (either with `git clone --recursive` or `git submodule --init --update` afterwards), build and run it:
+Simply clone this repository recursively
+
+```shell
+git clone --recursive REPOSITORY_URL
```

Comment:
does this `REPOSITORY_URL` get autoreplaced on render? otherwise it should be the URL of this repo, so users can copy&paste directly.